app/view/panel/application/StartViewController.js

```diff
@@ -30,14 +41,18 @@ Ext.define('MoMo.admin.view.panel.application.StartViewController', {
                 })
             ],
             view: new ol.View({
-                center: ol.proj.fromLonLat([7.1067575, 50.7374741]),
-                zoom: 12
+                center: ol.proj.fromLonLat([x, y]),
+                projection: ol.proj.get(projCode),
+                zoom: zoom
             })
         });
 
         view.down('gx_map').setMap(olMap);
     },
 
+    /**
+     * 
+     */
     addMapCrossHair: function() {
```

Comment:
It seems that this method is still called when the `boxready` event of the `gx_map` component is being fired. So i'll leave it there.